### PR TITLE
[SMALLFIX]Replace something with some real content 

### PR DESCRIPTION
**Pull request by starry-sky6688**
Replace {% include Configuring-Alluxio-with-NFS/underfs-address.md %} with the real content alluxio.underfs.address=/mnt/nfs

### CODE REVIEW

File: docs/cn/Command-Line-Interface.md [hunk 1]
```diff
@@ -91,7 +109,10 @@ Usage: alluxio fs [generic options]
 
 使用举例：使用`chgrp`命令能够快速修改一个文件的所属组：
 
-{% include Command-Line-Interface/chgrp.md %}
+```
```
Comment (apc999) [inline docs/cn/Command-Line-Interface.md]:
```suggestion
```properties
```

Comment (apc999) [reply docs/cn/Command-Line-Interface.md]: properties should be the same line as ```

**Comment by starry-sky6688** [comment]
alluxio-bot, check this please

**Comment by newnius** [comment]
Jenkins, test this please

**Comment by starry-sky6688** [comment]
Jenkins, test this please

**Comment by starry-sky6688** [comment]
@apc999 I have removed properties to the same line as '''

**Comment by starry-sky6688** [comment]
Jenkins, test this please

**Comment by starry-sky6688** [comment]
Jenkins, test this please

**Comment by starry-sky6688** [comment]
Jenkins, test this please

**Pull request closed without merging**